chart/byconity/files/cnch-config.yaml

```diff
@@ -26,7 +26,6 @@ catalog_service:
   type: fdb
   fdb:
     cluster_file: /etc/byconity/fdb/cluster-file
-hdfs_addr: xx
```

Comment:
I think we have to keep hdfs_addr in cnch-config.xml and remove them in the other places

Reply:
When both hdfs_address and hdfs_ha_nameservice are present, hdfs_address will be used preferentially and hdfs_ha_nameservice will not be used, so I want to make a choice from the outside.

Reply:
I think we can also put `hdfs_ha_nameservice` and all other hdfs config in here to avoid code duplication